lib/linguist/languages.yml

```diff
@@ -1772,6 +1772,7 @@ GraphQL:
   type: data
   extensions:
   - ".graphql"
+  - ".graphqls"
```

Comment:
Extensions should be sorted, except for the first one, so `.graphqls` should come last.